google/cloud/storage/internal/grpc/metrics_exporter_options_test.cc

```diff
@@ -28,6 +34,16 @@ namespace storage_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace {
 
+using ::google::cloud::testing_util::IsProtoEqual;
+using ::google::protobuf::TextFormat;
+using ::testing::Contains;
+using ::testing::ContainsRegex;
+using ::testing::Pair;
+
+auto EmptyResource() {
+  return opentelemetry::sdk::resource::Resource::Create({});
```

Comment:
nit: This is not actually empty. This is `opentelemetry::sdk::resource::Resource::GetDefault()` which has things like the otel library version.

Not that it matters, but I think you want: `opentelemetry::sdk::resource::Resource::GetEmpty()`

https://github.com/open-telemetry/opentelemetry-cpp/blob/ce14bf6520a059ff0bca81df45603722d965ad7c/sdk/include/opentelemetry/sdk/resource/resource.h#L56

Reply:
Changed to `TestResource()`.